docs/guides/create-interface.md

```diff
@@ -0,0 +1,62 @@
+---
+id: create-interface
+title:  Create an Interface
+---
+
+## **Introduction**
+
+In this guide, we'll walk you through creating your own interface
+
+As always, if you need any help, message us on [Discord](https://discord.com/invite/Z5m88a5qWu)!
+
+## **Prerequisites**
+
+You'll need the following installed before building your plugin:
+
+- `nvm`
+- `yarn`
+
+## **Getting started**
+
+Use the following command to spin up a project folder for your plugin to get started.
+
+```
+npx @web3api/cli create api interface <project-name>
+```
+
+## **Overview of project files**
+
+Your project should look something like this:
+
+```
+src/
```

Comment:
This folder structure is not accurate. If you run the `w3 create` command above you can see what the expected folder structure is.